samples/simulation/gaussian-initial-state/Program.qs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.GaussianPreparation {
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic; 
+    open Microsoft.Quantum.Arrays;
+
+    @EntryPoint()
+    operation RunProgram(recursive : Bool, nQubits : Int) : Unit {
+        let stdDev = IntAsDouble(2 ^ nQubits) / 6.;
+        let mean = IntAsDouble(2 ^ (nQubits - 1)) - 0.5;
+        use register = Qubit[nQubits];
+        // Call the recursive implementation.
+        if recursive {
+            PrepareGaussianWavefunctionRecursive(stdDev, mean, nQubits, [], register);
+        } else {
+            PrepareGaussianWavefunction(stdDev, mean, register);
+        }
+        // Output the resulting quantum state.
+        DumpRegister((), register);
```

Comment:
I'd suggest to add a comment explaining why the first argument is an empty tuple as this might confuse people

Reply:
We've used that fairly universally (mentioned in the API docs for DumpRegister, though that could be a good place to add more examples), but happy to add a comment here as well.